src/tools/rbd_mirror/Mirror.cc

```diff
@@ -147,6 +160,13 @@ class MirrorAdminSocketHook : public AdminSocketHook {
     if (r == 0) {
       commands[command] = new FlushCommand(mirror);
     }
+
+    command = "rbd mirror release leader";
```

Comment:
Nit: perhaps "rbd mirror leader release" in case new leader asok commands are added in the future